app/views/bookings/show.html.erb

```diff
@@ -1,14 +1,28 @@
+
+<div class="container">
 <% if policy(@booking).show? %>
-<h1>Booking for <%= link_to @dog[:name], dog_path(@dog) %> and <%= link_to @user[:name], user_path(@user) %></h1>
+  <div class="card-show">
 
-<%= cl_image_tag @dog.photo, width: 200, height: 150, crop: :fill %>
-<p>Description: <%= @booking[:description] %></p>
-<p>Date: <%= @booking[:date] %></p>
-<p>Confirmed?: <%= @booking[:confirmed] ? 'yes' : 'no' %></p>
 
-<%= link_to "Back to #{@dog[:name]}'s page", dog_path(@dog) %>
-<% end %>
+    <h1>Booking for <%= link_to @dog[:name], dog_path(@dog), class:"links-sessions" %> and <%= link_to @user[:name], user_path(@user), class:"links-sessions" %></h1>
```

Comment:
Something seems to be missing here

Reply:
<img width="1024" alt="Screenshot 2019-08-14 at 22 43 41" src="https://user-images.githubusercontent.com/51639218/63054918-1031d400-bee5-11e9-940f-55d6d045b750.png">
if it regards the show of booking then it has only been moved elsewhere but the link is still here

Reply:
@Bcavez

Reply:
On this line half of the link to the dog page is missing. So either remove it entirely or fix.

Reply:
nevermind